packages/commonwealth/client/scripts/controllers/server/threads.ts

```diff
@@ -516,7 +517,90 @@ class ThreadsController {
       },
     });
   }
+  /**
+   * Add a set of links to a Thread
+   * @param args 
+   * @returns updated Thread
+   */
+  public async addLinks (args: {
+    threadId: number;
+    links: link[];
+  }): Promise<Thread>{
+    const response = await $.ajax({
```

Comment:
please do not use jquery for API calls, use axios instead (everywhere)

Reply:
@masvelio Good catch. Just copied the existing calls in this file with new links, but I'll convert the new ones to axios